shared/src/packages/users/libs/validation-schemas/user-sign-in.validation-schema.ts

```diff
@@ -7,17 +7,17 @@ const userSignIn = joi.object<UserSignInRequestDto, true>({
   email: joi
     .string()
     .trim()
-    .email({
-      tlds: {
-        allow: false,
-      },
-    })
+    .regex(/^[\w%+.-]+@[\d.A-Za-z-]+\.[A-Za-z]{2,}$/)
     .required()
     .messages({
-      'string.email': UserValidationMessage.EMAIL_WRONG,
+      'string.pattern.base': UserValidationMessage.EMAIL_WRONG,
       'string.empty': UserValidationMessage.EMAIL_REQUIRE,
     }),
-  password: joi.string().trim().required(),
+  password: joi.string().trim().min(4).max(20).required().messages({
```

Comment:
Create UserValidationRule enum and pass all `magic` values there (like min length value, max length value, regex for email) + you may use `UserValidationRule` enum both on UserValidationMessage (where you've also defined some magic numbers) and in validation schema